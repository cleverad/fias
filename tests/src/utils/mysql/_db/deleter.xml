<?xml version="1.0" ?>
<dataset>
    <deleter id="1" row1="row_1_1" row2="row_2_1" />
    <deleter id="2" row1="row_1_2" row2="row_2_2" />
    <deleter id="3" row1="row_1_3" row2="row_2_3" />
    <deleter id="4" row1="row_1_4" row2="row_2_4" />
</dataset>
