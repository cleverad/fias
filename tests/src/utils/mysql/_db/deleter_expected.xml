<?xml version="1.0" ?>
<dataset>
    <deleter id="2" row1="row_1_2" row2="row_2_2" />
</dataset>
